src/main/java/org/apache/mesos/scheduler/SchedulerUtils.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.mesos.scheduler;
+
+/**
+ * This class provides utilities common to the construction and operation of Mesos Schedulers.
+ */
+public class SchedulerUtils {
```

Comment:
This name could be more specific, or is there an expectation that more functionality will be added here later?